### PR TITLE
homework 1: task1, task2 and task3

### PR DESCRIPTION
**Pull request by AntTurieva**
homework 1: task1, task2 and task3

### CODE REVIEW

File: src/main/java/hw_1/Task_1.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package hw_1;
+
+public class Task_1 {
+    public static void main(String[] args)
+    {
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task_1.java]: this symbol "{" should be after  main(String[] args). So, "public static void main(String[] args) {"

File: src/main/java/hw_1/Task_1.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        String name = "Antonina";
+        int age = 23;
+        float weight = 50.5F;
+
+
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task_1.java]: You have a 3 extra spaces above. Please leave only one

File: src/main/java/hw_1/Task_1.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package hw_1;
+
+public class Task_1 {
+    public static void main(String[] args)
+    {
+        String name = "Antonina";
+        int age = 23;
+        float weight = 50.5F;
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task_1.java]: You can use "f" instead of "F"

File: src/main/java/hw_1/Task_2.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package hw_1;
+
+public class Task_2 {
+    public static void main(String[] args)
+    {
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task_2.java]: this symbol "{" should be after main(String[] args). So, "public static void main(String[] args) {"

File: src/main/java/hw_1/Task_3.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package hw_1;
+
+public class Task_3 {
+    public static void main(String[] args)
+    {
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task_3.java]: this symbol "{" should be after main(String[] args). So, "public static void main(String[] args) {"

File: src/main/java/hw_1/Task_3.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        int a =5;
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task_3.java]: Please add an extra space after "="

File: src/main/java/hw_1/Task_3.java [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+        int b =3;
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task_3.java]: Please add an extra space after "="

File: src/main/java/hw_1/Task_3.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+//if we can not use numbers
+
+        a = a + b; //5+3=8
+        b = a - b; //8-3=5
+        a = a - b; // 8-5=3
+
+        System.out.println(a);
+        System.out.println(b);
+
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_1/Task_3.java:16]: Please, remove all comments from the file

File: src/main/java/hw_1/Task_1.java (hunk 1 above)
Comment (AntTurieva) [reply src/main/java/hw_1/Task_1.java]: fixed

File: src/main/java/hw_1/Task_1.java (hunk 2 above)
Comment (AntTurieva) [reply src/main/java/hw_1/Task_1.java]: fixed

File: src/main/java/hw_1/Task_1.java (hunk 3 above)
Comment (AntTurieva) [reply src/main/java/hw_1/Task_1.java]: fixed

File: src/main/java/hw_1/Task_2.java (hunk 4 above)
Comment (AntTurieva) [reply src/main/java/hw_1/Task_2.java]: fixed

File: src/main/java/hw_1/Task_3.java (hunk 5 above)
Comment (AntTurieva) [reply src/main/java/hw_1/Task_3.java]: fixed

File: src/main/java/hw_1/Task_3.java (hunk 6 above)
Comment (AntTurieva) [reply src/main/java/hw_1/Task_3.java]: fixed

File: src/main/java/hw_1/Task_3.java (hunk 7 above)
Comment (AntTurieva) [reply src/main/java/hw_1/Task_3.java]: fixed

File: src/main/java/hw_1/Task_3.java (hunk 8 above)
Comment (AntTurieva) [reply src/main/java/hw_1/Task_3.java:16]: fixed

**Pull request merged**